llvm/test/tools/llvm-objcopy/ELF/cross-arch-headers.test

```diff
@@ -3,6 +3,7 @@
 # Note that we don't actually need any DWARF to produce the DWO file.
 
 # RUN: yaml2obj %s -o %t.o
+# RUN: yaml2obj -DFLAGS=[EF_AMDGPU_MACH_AMDGCN_GFX900] %s -o %t.o
```

Comment:
You're overwriting the older file here which is a bigger issue 
I now agree a separate test file is needed, so feel free to revert to the previous approach

Reply:
> I now agree a separate test file is needed, so feel free to revert to the previous approach

Please don't.